Formula/urler.rb

```diff
@@ -0,0 +1,21 @@
+class Urler < Formula
+  desc "Command-line tool for URL parsing and manipulation"
+  homepage "https://github.com/curl/urler"
+  url "https://github.com/curl/urler/archive/64a48c004cbc0e91d15aff2e090f423ab7555b6c.tar.gz"
+  version "0.1"
```

Comment:
We don't build from arbitrary commits. Please ask upstream if they'll consider tagging releases in their repository.

Reply:
Thanks as @ZhongRuoyu just commented version 0.1 is not released yet, this PR is a bit too green.